src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/serviceprovider/performance/PerformanceReportJobListener.kt

```diff
@@ -14,6 +17,10 @@ import kotlin.io.path.createTempDirectory
 @Component
 class PerformanceReportJobListener(
   private val s3Service: S3Service,
+  private val emailSender: EmailSender,
+  @Value("\${notify.templates.service-provider-performance-report-ready}") private val notifyTemplateId: String,
+  @Value("\${interventions-ui.baseurl}") private val interventionsUiBaseUrl: String,
```

Comment:
Do you need to add `interventions-ui.baseurl` var to application.yml?
 (Or I guess would go in values-{env}.yaml because it will be different for each env.)

Reply:
yes that's right - the var is defined per environment. it's already there since we use it for generating other email / delius links.